packages/smooth_app/lib/pages/onboarding/welcome_page.dart

```diff
@@ -0,0 +1,217 @@
+import 'package:smooth_ui_library/util/ui_helpers.dart';
+
+/// Welcome page for first time users.
+class WelcomePage extends StatelessWidget {
+  const WelcomePage();
+
+  @override
+  Widget build(BuildContext context) {
+    final Size screenSize = MediaQuery.of(context).size;
+    // Side padding is 8% of total width.
+    final double sidePadding = screenSize.width * 8 / 100;
+    // Top padding is 16% of total width.
+    final double topPadding = screenSize.height * 16 / 100;
+    // Bottom padding is 4% of total width.
+    final double bottomPadding = screenSize.height * 4 / 100;
+    final AppLocalizations appLocalizations = AppLocalizations.of(context)!;
+    final TextStyle headlineStyle =
+        Theme.of(context).textTheme.headline2!.apply(color: Colors.white);
+    final TextStyle largeButtonTextStyle =
+        Theme.of(context).textTheme.headline3!.apply(color: Colors.white);
+    final TextStyle bodyTextStyle =
+        Theme.of(context).textTheme.bodyText1!.apply(color: Colors.white);
+    return Scaffold(
+      body: Padding(
+        padding: EdgeInsets.only(
+            top: topPadding,
+            bottom: bottomPadding,
+            left: sidePadding,
+            right: sidePadding),
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: [
+            Text(appLocalizations.whatIsOff, style: headlineStyle),
+            Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                Padding(
+                  padding: const EdgeInsets.only(left: SMALL_SPACE),
+                  child: Text(
+                    appLocalizations.country_label,
+                    style: bodyTextStyle,
+                  ),
+                ),
+                const CountrySelector(),
+                Padding(
+                  padding: const EdgeInsets.only(left: SMALL_SPACE),
+                  child: Text(
+                    appLocalizations.country_selection_explanation,
+                    style: bodyTextStyle,
+                  ),
+                ),
+              ],
+            ),
+            Row(
+              children: [
+                Expanded(
+                  child: TextButton(
+                    style: TextButton.styleFrom(
+                      backgroundColor: SmoothTheme.getColor(
+                        Theme.of(context).colorScheme,
+                        SmoothTheme
+                            .MATERIAL_COLORS[SmoothTheme.COLOR_TAG_BLUE]!,
+                        ColorDestination.BUTTON_BACKGROUND,
+                      ),
+                      shape: RoundedRectangleBorder(
+                          borderRadius: BorderRadius.circular(SMALL_SPACE)),
+                      primary: Colors.white,
+                    ),
+                    onPressed: () {
+                      // TODO(jasmeet): Navigate to the next page.
+                    },
```

Comment:
What happens here? In a first approach you could store the country in the preferences and go to the home page, I guess.

Reply:
we should go to the next onbpoarding page, but good point for now.